lib/DDG/Spice/Sentencer.pm

```diff
@@ -0,0 +1,32 @@
+package DDG::Spice::Sentencer;
+
+# ABSTRACT: Output a random number of metaphor sentences.
+
+use strict;
+use DDG::Spice;
+
+# spice proxy_cache_valid => "418 1d";
```

Comment:
I think we may want to uncomment this here... Users will want new results each time to mimic the API, so I don't think caching is necessary.